src/firebase/api.jsx

```diff
@@ -212,7 +214,30 @@ function setGameStateListen(gameID, setGameState, listeners, setListeners) {
 			setGameState(snapshot.val());
 		},
 	);
-	setListeners({ ...listeners, stateListener });
+	setListeners({ ...listeners, stateListener: stateListener });
+}
+
+function setRoundsListen(
+	gameID,
+	setRoundState,
+	listeners,
+	setCurrentRound,
+	setCurrentTrial,
+	setListeners,
+) {
+	const stateListener = onValue(
+		ref(database, "/games/" + gameID + "/rounds"),
```

Comment:
rather than have it listen to the whole thing like this, we can remove and renew a listener to the votes section of a current round and current trial. Tho this also works since the data being sent back and forth is pretty small.